page-objects/src/components/bottomBar/ProblemsView.ts

```diff
@@ -61,6 +71,14 @@ export class ProblemsView extends AbstractElement {
         }
         return markers;
     }
+
+    /**
+     * Gets the count badge
+     * @returns Promise resolving to the WebElement representing the count badge
+     */
+    async getCountBadge(): Promise<WebElement> {
```

Comment:
Added `getCountBadge()`

Reply:
Could you please add another function which would return Promise<number>.

Reply:
@mlorinc do you mean like so?
```
    /**
     * Gets the count badge
     * @returns Promise resolving to the WebElement representing the count badge
     */
    async getCountBadgeAsync(): Promise<WebElement> {
        return await this.findElement(ProblemsView.locators.ProblemsView.changeCount);
    }

    /**
     * Gets the count badge
     * @returns Promise resolving to the WebElement representing the count badge
     */
     getCountBadge(): Promise<WebElement> {
        return this.findElement(ProblemsView.locators.ProblemsView.changeCount);
    }
```
I could make this change, but every other method within ProblemsView is in the form of
```
    async methodName: Promise<WebElement> {
        ...
    }
```

Reply:
@mlorinc could you please answer here? thank you